R/pkg/R/mllib.R

```diff
@@ -66,8 +67,9 @@ setClass("KMeansModel", representation(jobj = "jobj"))
 #'               \url{https://stat.ethz.ch/R-manual/R-devel/library/stats/html/family.html}.
 #' @param tol Positive convergence tolerance of iterations.
 #' @param maxIter Integer giving the maximal number of IRLS iterations.
-#' @return a fitted generalized linear model
+#' @return \code{spark.glm} returns a fitted generalized linear model
```

Comment:
since this is the page for `spark.glm` I think it's ok if it simply says `returns a fitted...`
it seems a bit odd to reference `spark.glm` again.